src/math/log_10.c

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: BSD-3-Clause/
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+//
+//
+
+/* Include Files */
+#include <sof/math/log_10.h>
+
+/* Defines Constant*/
+#define BASE10LOG_DENOM_WORDLENGTH 0x1F
+#define BASE10LOG_WRAP_SCHAR_BITS 0xFF
+#define BASE10LOG_NUMER_WORDLENGTH 0x3
+#define BASE10LOG_UPPERBYTES 0xFFFFFF
+#define BASE10LOG_NUM_SHIFTBITS 0x1C
+/* Function Declarations */
+static int64_t division(int64_t numerator, int64_t denominator);
+
+/* Function Definitions */
+/*
+ * Arguments	: int64_t numerator
+ *		  int64_t denominator
+ * Return Type	: int64_t
+ */
+static int64_t division(int64_t numerator, int64_t denominator)
```

Comment:
What does this do ? Is it a 64bit division ?

Reply:
removed division ()